components/icons/EditIcon.tsx

```diff
@@ -1,9 +1,13 @@
-const EditIcon: React.FC = ({}) => (
+interface props {
+  color: string;
+}
+
+const EditIcon: React.FC<props> = ({ color }) => (
```

Comment:
Prefer title case for interface names.
```suggestion
interface Props {
  color: string;
}

const EditIcon: React.FC<Props> = ({ color }) => (
```